### ISSUE TITLE
Accessibility — font size and display size (Android and Windows)

### ISSUE THREAD

**Issue by testuser51** [tootsuite/mastodon#14470]
### Expected behaviour
The site is adapted for visually impaired people or the site ignores the settings

### Actual behaviour
Not enough space for text or unable to use site
<img width="540" alt="555" src="https://user-images.githubusercontent.com/53906414/88814622-0622e700-d1c3-11ea-8fb9-1496259e12db.png">
<img width="827" alt="333" src="https://user-images.githubusercontent.com/53906414/88814650-120ea900-d1c3-11ea-9df3-51c999bb073d.png">

### Steps to reproduce the problem
**Android**'s 10 settings: Accessibility - font size and display size = largest
Browser's settings (Firefox 79): Accessibility  - automatic font sizing
![25](https://user-images.githubusercontent.com/53906414/88814087-59486a00-d1c2-11ea-85fc-74bb75d2a374.jpg)

**Windows**: 
Browser: Firefox 79.0
Options: Language and Appearance — Size 30
![image](https://user-images.githubusercontent.com/53906414/88814818-44b8a180-d1c3-11ea-826f-8b3715a88d2e.png)

### Specifications
Mastodon v3.2.0

**Comment by testuser51** [comment]
Android: Less important, but not cool
![7474](https://user-images.githubusercontent.com/53906414/88816788-73377c00-d1c5-11ea-9284-9247103452fd.jpg)

**Comment by Sasha-Sorokin** [comment]
Regarding the menu links, I believe we should simplify it to include much fewer links and hide additional ones under submenu (covered in #12334), other than that these are valid issues, but it's questionable if some of them can be fixed, keep this open, though.

Related: #14210, #873.

**Comment by mgifford** [comment]
Providing URLs is helpful. 

WCAG 2.0 specifies 200%. Some users zoom in much more than this.

I had trouble replicating this on Mastodon.social
